package.json

```diff
@@ -69,6 +69,7 @@
     "cloudgov-style": "^2.6.5",
     "codecov.io": "^0.1.6",
     "css-loader": "^0.23.0",
+    "date-fns": "^2.0.0-alpha.7",
```

Comment:
would prefer to use prod ready packages (i.e. no alpha)

Reply:
No worries.